include/itkDataManipulation.h

```diff
@@ -1,119 +0,0 @@
-/*=========================================================================
```

Comment:
I assume these file deletions should have been in a previous commit? Also it might make sense to squash commits 2 and 3, to avoid appearance of `itkDataManipulation.h` only to be immediately renamed.